homeassistant/components/tractive/__init__.py

```diff
@@ -142,7 +152,15 @@ async def _listen(self):
                 continue
 
     def _send_hardware_update(self, event):
-        payload = {ATTR_BATTERY_LEVEL: event["hardware"]["battery_level"]}
+        payload = {
+            ATTR_BATTERY_LEVEL: event["hardware"]["battery_level"],
+            ATTR_LIVE_TRACKING_REMAINING: event.get("live_tracking", {}).get(
```

Comment:
Are you sure we have to use `get()` for `live_tracking`? I suppose it is always present in the hardware event.

Reply:
After 8h of catching events, I have one without `live_tracking`:
```json
{"message": "tracker_status", "tracker_id": "FEBTCACU", "tracker_state": "OPERATIONAL", "tracker_state_reason": "POWER_SAVING", "position": {"time": 1630510307, "latlong": [0, 0], "sensor_used": "GPS", "accuracy": 17, "speed": 2.5, "course": 3, "altitude": 63, "time_rcvd": 1630510310}, "hardware": {"time": 1630488774, "battery_level": 96, "temperature_state": "NORMAL", "power_saving_zone_id": "611d4497d5444b0432669998", "clip_mounted_state": null}, "charging_state": "NOT_CHARGING", "battery_state": "FULL", "power_saving_zone_id": "611d4497d5444b0432669998"}
```

Sorry, my bad. However, the `tracker_state` was always present in the events.